toml/_parser.ts

```diff
@@ -134,6 +134,7 @@ function failure(): Failure {
   return { ok: false };
 }
 
+// deno-lint-ignore deno-style-guide/exported-function-args-maximum
 export function unflat(
```

Comment:
This function is only exported as internal util and not part of public API. Can we exclude this from lint target?